tests/shared/mount_functions.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+function ssgts_mount() {
```

Comment:
Function similar to this already exists in `tests/shared/partition.sh` - `mount_bind_partition`